### PR TITLE
updates to V&V table, to the adherence and treatments based on Greg's…

### PR DESCRIPTION
**Pull request by SylLutze**
… feedback

### CODE REVIEW

**Review by aflaxman — APPROVED** [review]

**Review by plinds — APPROVED** [review]

File: docs/source/concept_models/vivarium_us_cvd/concept_model.rst [hunk 1]
```diff
@@ -418,18 +409,27 @@ First, it is determined if the simulant will have a healthcare interaction in th
...
+    - 
   * - B
-    - LDL-C measreument error pulled from a normal distribution with mean=0 and SD=3 mg/dL    
+    - LDL-C measreument error pulled from a normal distribution with mean=0 and SD=0.08 mmol/L    
     - [McCormack_2020]_
   * - C
-    - 19.4% will not start medication; 14.2% will receive high intensity statin; 55.1% medium intensity; and 11.3% low intensity 
-    - [Morales_2018]_ [Arnett_2019]_
+    - If simulant is in the acute or post MI or stroke states  
+    - [Morales_2018]_ [Arnett_2019]_ [Nguyen_2015]_
```
Comment (alibow) [inline docs/source/concept_models/vivarium_us_cvd/concept_model.rst:419]: Is there an error here? I Don't see instructions here just "if simulant is in...". Just confirming!

File: docs/source/concept_models/vivarium_us_cvd/concept_model.rst [hunk 2]
```diff
@@ -559,9 +555,18 @@ Treatment Effects
 
 **Blood Pressure Treatments**  
 
+.. todo::
+  - Add parameter variation if needed to SBP  
+
 Blood pressure treatment efficacy is dependent on a simulant's SBP value. Full efficacy data is here:
 /share/scratch/projects/cvd_gbd/cvd_re/simulation_science/drug_efficacy_sbp_new.csv [Law_2009]_
 
+For each seed/draw, a parameter value for efficacy will be selected based on table above. While we plan
```
Comment (alibow) [inline docs/source/concept_models/vivarium_us_cvd/concept_model.rst:564]: I think we typically just vary parameter uncertainty just at the draw level, not draw/seed level.

File: docs/source/concept_models/vivarium_us_cvd/concept_model.rst [hunk 3]
```diff
@@ -559,9 +555,18 @@ Treatment Effects
...
 **Blood Pressure Treatments**  
 
+.. todo::
+  - Add parameter variation if needed to SBP  
+
 Blood pressure treatment efficacy is dependent on a simulant's SBP value. Full efficacy data is here:
 /share/scratch/projects/cvd_gbd/cvd_re/simulation_science/drug_efficacy_sbp_new.csv [Law_2009]_
 
+For each seed/draw, a parameter value for efficacy will be selected based on table above. While we plan 
+to add a more formal variation parameter to the table, please allow an up to 5% variation on value for each seed/draw.
```
Comment (alibow) [inline docs/source/concept_models/vivarium_us_cvd/concept_model.rst:565]: I think we need to specify the distribution within the bounds of the +/- 5% as well. Uniform? Normal?

File: docs/source/concept_models/vivarium_us_cvd/concept_model.rst [hunk 4]
```diff
@@ -573,14 +578,19 @@ SBP decrease = SBP treatment efficacy * Adherence score
 
 **LDL-C Treatments** 
 
-LDL-C treatment efficacy is a percent reduction in LDL-C level. This means that simulants with higher 
+LDL-C treatment efficacy is a **percent reduction** in LDL-C level. This means that simulants with higher 
 initial LDL-C levels will see a higher total reduction. The full efficacy data is here: 
-/share/scratch/projects/cvd_gbd/cvd_re/simulation_science/drug_efficacy_ldl.csv [Hou_2009]_ [Goff_2014]_
+/share/scratch/projects/cvd_gbd/cvd_re/simulation_science/drug_efficacy_ldl.csv [Law_2003]_ [Goff_2014]_ [Descamps_2015]_
+
+For each seed/draw, a parameter value for efficacy will be selected based on the mean and 95% confidence
```
Comment (alibow) [inline docs/source/concept_models/vivarium_us_cvd/concept_model.rst:585]: Same comments as above!

File: docs/source/concept_models/vivarium_us_cvd/concept_model.rst (hunk 1 above)
Comment (SylLutze) [reply docs/source/concept_models/vivarium_us_cvd/concept_model.rst:419]: This is probably not a needed line in the table

File: docs/source/concept_models/vivarium_us_cvd/concept_model.rst (hunk 3 above)
Comment (SylLutze) [reply docs/source/concept_models/vivarium_us_cvd/concept_model.rst:565]: Makes sense, added

**Pull request merged**